packages/components/src/DateTimePickers/pickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -173,6 +187,19 @@ class DateTimePicker extends React.Component {
 				aria-label="Date picker"
 			>
 				{viewElement}
+				<div className={theme.footer}>
+					<Action
+						label={this.props.t('DATEPICKER_TODAY_ACTION', {
+							defaultValue: 'Today',
+						})}
+						aria-label={this.props.t('DATEPICKER_SELECT_TODAY', {
```

Comment:
Nice one :+1: but you can improve it by describing a few more _Pick today as selected date_ or something like that

Reply:
Sorry, I don't get you.
Do you mean set today as default?

Reply:
I suggest you improve the aria-label value by choosing _Pick today as selected date_ instead